### PR TITLE
Feature : Typography

### LINKED ISSUE #53: Typography
Create the DS's type, confirm all font family are as expected while matching the designs

### LINKED ISSUE #60: Text components

### PR DESCRIPTION
**Pull request by MamboBryan**
name: Pull request
about: Create a pull request
label: 'triage me'
Fixes #53
Fixes #60 

---

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Ensure your branch is up to date with develop branch.
- [ ] Ensure your branch runs without breaking.
- [ ] Ensure you solved any merge conflicts.
- [ ] Smaller PRs for the ease of reviewing
- [ ] Make sure to open a GitHub issue as a bug/feature request before writing your code! This means that we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass (`./gradlew --init-script gradle/init.gradle.kts spotlessApply` to automatically apply formatting)
- [ ] Appropriate docs were updated (You are adding code so...)
- [ ] Tests have been written

Is this your first Pull Request?
- [ ] Run `./tools/setup.sh`
- [ ] Import the code formatting style as explained in [the setup script](/tools/setup.sh#L40).

Fixes #<issue_number_goes_here> 🦕

# Screenshot
> For PRs that change the UI, attach a before/after screenshot.
> **[REMOVE THIS SECTION IF IT DOES NOT APPLY TO THIS PR]**

### CODE REVIEW

File: chai/src/main/java/com/droidconke/chai/typography/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Typography
+This is a simple doc that should be used in line with the typography tokens in the design system
```
Comment (tamzi) [inline chai/src/main/java/com/droidconke/chai/typography/README.md:2]: Replace with: 
These are the typography tokens in the design system.

File: chai/src/main/java/com/droidconke/chai/typography/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Title
+This is the **ONLY** token to use for the main purpose of a screen, and is the topmost hierarchy in any screen.
+
+> It should only be used or appear once in a screen
+
+Example: 
+If you have an app with two screens, characters and character, where characters shows a list of people and character shows the details of one person.
+You can use title for Characters and Character .
+
```
Comment (tamzi) [inline chai/src/main/java/com/droidconke/chai/typography/README.md:19]: Missing code sample.

File: chai/src/main/java/com/droidconke/chai/typography/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- display
+- paragraph
+- body
+- label
+
+Usage
+
+### Display
+This is the token to use when you want to highlight or make any content stand out. It should not be used as a header, title or body text.
+
```
Comment (tamzi) [inline chai/src/main/java/com/droidconke/chai/typography/README.md:37]: Missing code sample. for the others

File: chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+import androidx.compose.ui.text.style.LineHeightStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.em
+import androidx.compose.ui.unit.sp
+import com.droidconke.chai.ChaiTheme
+import com.droidconke.chai.atoms.MontserratSans
+import com.droidconke.chai.utils.ChaiPreview
+
+private val DUMMY_CHAI_TEXT = "Design systems provide guidelines and reusable components to ensure consistent and efficient design across digital products, improving user experience and streamlining team workflows."
+
```
Comment (tamzi) [inline chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt:18]: Would it make sense to use  an inline doc here... showing how we use the specific composable

File: chai/src/main/java/com/droidconke/chai/components/CText.kt [hunk 5]
```diff
@@ -27,8 +27,7 @@ import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.unit.sp
 import com.droidconke.chai.atoms.*
-import com.droidconke.chai.atoms.MontserratRegular
-import com.droidconke.chai.atoms.MontserratThin
+import com.droidconke.chai.atoms.MontserratSans
 import com.droidconke.chai.chaiColorsPalette
 
 /**
```
Comment (tamzi) [inline chai/src/main/java/com/droidconke/chai/components/CText.kt:33]: Is this inline doc up to date?

**Review by tamzi — CHANGES_REQUESTED** [review]

File: chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+        Text(text = DUMMY_CHAI_TEXT, style = ChaiLabelTextStyle)
+    }
+}
+
+data class ChaiTypography(
+    val display: TextStyle = ChaiDisplayTextStyle,
+    val title: TextStyle = ChaiTitleTextStyle,
+    val header: TextStyle = ChaiHeaderTextStyle,
+    val paragraph: TextStyle = ChaiParagraphTextStyle,
+    val body: TextStyle = ChaiBodyTextStyle,
```
Comment (janewaitara) [inline chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt:127]: What's the difference between paragraph and body? Can we just have one as the use case seem like it's the same

File: chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+    val label: TextStyle = ChaiLabelTextStyle
```
Comment (janewaitara) [inline chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt:128]: Why isn't there a button text?

File: chai/src/main/java/com/droidconke/chai/typography/README.md (hunk 1 above)
Comment (MamboBryan) [reply chai/src/main/java/com/droidconke/chai/typography/README.md]: Done...

File: chai/src/main/java/com/droidconke/chai/typography/README.md (hunk 2 above)
Comment (MamboBryan) [reply chai/src/main/java/com/droidconke/chai/typography/README.md:19]: Done...

File: chai/src/main/java/com/droidconke/chai/typography/README.md (hunk 3 above)
Comment (MamboBryan) [reply chai/src/main/java/com/droidconke/chai/typography/README.md:56]: Done...

**Review by janewaitara — APPROVED** [review]

File: chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt
Comment (tamzi) [inline chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt:1]: waiting. @MamboBryan  :)

**Review by tamzi — CHANGES_REQUESTED** [review]

**Review by tamzi — APPROVED** [review]

**Pull request merged**